introduction_to_amazon_algorithms/jumpStart_tabular_classification/Amazon_JumpStart_Tabular_Classification_LightGBM_CatBoost.ipynb

```diff
@@ -0,0 +1,597 @@
+{
```

Comment:
<p>For the screenshot, it's preferred to display a standalone JPG/PNG from the notebook, rather than embedding the image directly in the notebook JSON. Alternatively, you could use a markdown table to show the data preview instead of an image.</p><p>(This applies to all 4 MNIST/Abalone screenshots across the 4 notebooks in this PR)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
@Neo9061 Here's an example of a notebook referring to png images in a sibling folder: https://github.com/aws/amazon-sagemaker-examples/blob/main/hyperparameter_tuning/rapids_bring_your_own/rapids_sagemaker_hpo.ipynb